.ci/e2e/scripts/tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd tests
+pytest --everest-prefix ../dist core_tests/startup_tests.py
```

Comment:
There is currently no results artifact generation being done here.

I would advise to add this, even if it does currently only provide basic error output. This makes report retrieval much easier and can later be expanded to include other results (as discussed: build-logs and other reports).